Lighting Board/Code/Examples/Lighting_Board/Lighting_Board.ino

```diff
@@ -0,0 +1,152 @@
+
+  
+  autonomy.begin();
+  autonomy.setBrightness(BRIGHTNESS);
+  pinMode(HEADLIGHT_TOGGLE, OUTPUT);
+  digitalWrite(HEADLIGHT_TOGGLE, LOW);
+}
+
+void loop()
+{
+  packet = RoveComm.read();
+  if (packet.data_id != 0)
+  {
+    switch (packet.data_id)
+      {
+        case RC_MULTIMEDIABOARD_HEADLIGHTINTENSITY_DATA_ID:
+        {
+          uint8_t* headlightEnable = (uint8_t*)packet.data;
+          if (headlightEnable > 0)
+            digitalWrite(HEADLIGHT_TOGGLE, HIGH);
+          else
+            digitalWrite(HEADLIGHT_TOGGLE, LOW);
+          break;
+        }
+        case RC_MULTIMEDIABOARD_LEDRGB_DATA_ID:
+        {
+          uint8_t* LEDRGB = (uint8_t*)packet.data;
+          for( uint16_t i=0; i<AUTONOMY_COUNT; i++)
+          {
+            autonomy.setPixelColor(i, autonomy.Color(LEDRGB[0], LEDRGB[1], LEDRGB[2]));
+          }
+          autonomy.show();
+          break;
+        }
+        case RC_MULTIMEDIABOARD_LEDPATTERNS_DATA_ID:
+        {
+          uint8_t* LEDPattern = (uint8_t*)packet.data;
+          switch (LEDPattern[0])
+          {
+            case MRDT:
+              for(uint16_t i=0; i<256; i++)
+              {
+                autonomy.setPixelColor(i, LOGO[i]);
+              }
+              autonomy.show();
+              break;
+            case BELGIUM:
+              for(uint16_t i=0; i<256; i++)
+              {
+                autonomy.setPixelColor(i, BELGIUM_FLAG[i]);
+              }
+              autonomy.show();
+              break;
+            case MERICA:
+              for(uint16_t i=0; i<256; i++)
+              {
+                autonomy.setPixelColor(i, FLAG[i]);
+              }
+              autonomy.show();
+              break;
+            case DIRT:
+              for(uint16_t i=0; i<256; i++)
+              {
+                autonomy.setPixelColor(i, DIRT_BLOCK[i]);
+              }
+              autonomy.show();
+              break;
+            case DOTA:
+              for(uint16_t i=0; i<256; i++)
+              {
+                autonomy.setPixelColor(i, DOTES[i]);
+              }
+              autonomy.show();
+              break;
+            case MCD:
+              for(uint16_t i=0; i<256; i++)
+              {
+                autonomy.setPixelColor(i, MCDONALD[i]);
+              }
+              autonomy.show();
+              break;
+            case WINDOWS:
+              for(uint16_t i=0; i<256; i++)
+              {
+                autonomy.setPixelColor(i, MSW[i]);
+              }
+              autonomy.show();
+              break;
+          }
+          break;
+        }
+        case RC_MULTIMEDIABOARD_STATEDISPLAY_DATA_ID:
+        {
+          uint8_t* displayState = (uint8_t*)packet.data;
+          switch (displayState[0])
+            {
+              case TELEOP:
+                for( uint16_t i=0; i<AUTONOMY_COUNT; i++)
+                {
+                  autonomy.setPixelColor(i, autonomy.Color(0,0,255));
```

Comment:
nit: comment the color that you are setting it to (yes its RGB but it's just more clear)